src/frontend/components/UI/WindowControls/index.scss

```diff
@@ -1,9 +1,11 @@
 .windowControls {
-  position: fixed;
-  top: 0;
-  right: 0;
+  grid-area: content;
+  justify-self: right;
+  align-self: start;
   z-index: 100;
   -webkit-app-region: no-drag;
+  position: sticky;
+  top: 0px;
```

Comment:
the windows controls were overlapping the sidebar in RTL languages

![image](https://github.com/user-attachments/assets/4e4af73b-8319-4065-8f39-5f05b613209b)

and now:
![image](https://github.com/user-attachments/assets/a4bd900a-967e-4d89-8340-da7f479c2523)